tests/unit/eblob_wrapper.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <boost/filesystem.hpp>
+
+#include <library/blob.h>
+
+#include "library/crypto/sha512.h"
+
+#include <eblob/eblob.hpp>
```

Comment:
`#include <...>` for global headers and `#include "..."` for local headers. Global headers should be placed before local. Both should be sorted.

Reply:
done

Reply:
done